src/app/public/modules/modal/modal.component.scss

```diff
@@ -72,6 +72,8 @@
 
 
 .sky-modal-header-buttons {
+  flex-shrink: 0.0001;
```

Comment:
It would be ideal to have a visual test that demonstrates this working.

Reply:
Added the test. Had to remove the baseline screenshots since there is a new `Open modal with extended title` button in the background that gets picked up in each modal screenshot.